tests/langchain/test_langchain_model_export.py

```diff
@@ -2640,71 +2640,79 @@ def fake_chat_stream_model():
     Version(langchain.__version__) < Version("0.0.311"), reason="feature not existing"
 )
 def test_simple_chat_model_stream_inference(fake_chat_stream_model):
-    with mlflow.start_run():
-        model_info = mlflow.langchain.log_model(fake_chat_stream_model, "model")
-
-    loaded_model = mlflow.pyfunc.load_model(model_info.model_uri)
-
     input_example = {
         "messages": [
             {"role": "system", "content": "You are a helpful assistant."},
             {"role": "assistant", "content": "What would you like to ask?"},
             {"role": "user", "content": "Who owns MLflow?"},
         ]
     }
+    signature = infer_signature(model_input=input_example, model_output=None)
+    with mlflow.start_run():
+        model_info = mlflow.langchain.log_model(
+            fake_chat_stream_model, "model",
+        )
 
-    chunk_iter = loaded_model.predict_stream(input_example)
-
-    finish_reason = None if Version(langchain.__version__) < Version("0.1.0") else "stop"
-
-    with mock.patch("time.time", return_value=1677858242):
-        chunks = list(chunk_iter)
-
-        for chunk in chunks:
-            assert "id" in chunk, "chunk id is lost."
-            chunk["id"] = None
+    with mlflow.start_run():
+        model_with_siginature_info = mlflow.langchain.log_model(
```

Comment:
Can we also add test for 1. log model with input_example but not signature? signature should be inferred from input_example; 2. spark udf if it's applicable?

Reply:
I think spark udf is not compatible with `predict_stream`, but this "log model with input_example but not signature?" is valuable to add

Reply:
I just checked, for this case "log model with input_example but not signature"
internally it just calls `_infer_signature_from_input_example` to infer signature and then log model with the signature. so it is equivalent with the signature I use in the test `signature = infer_signature(model_input=input_example, model_output=None)`.


But I found another issue, `_infer_signature_from_input_example` also tries to infer output schema from prediction result, but chat model output is incompatible with current inferring implementation. We can address it in follow-up PR :)